wpilib/wpilib/drive/differentialdrive.py

```diff
@@ -27,7 +29,7 @@ class DifferentialDrive(RobotDriveBase):
 
     Four motor drivetrain::
 
-        def robotInit(self):
+        def robotInit(self) -> None:
```

Comment:
I don't think this is really necessary. This should be kept as a simple example for teams.

Reply:
Hah, my find/replace did this. Good catch

Reply:
That's horrifying. You could've used something like pyannotate or something instead.

Reply:
Well with the amount of inconsistencies and fun python stuff we do, this was easier. And it worked great for the most part